packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -49,6 +55,11 @@ export abstract class InversifyHttpAdapter<
     this.#container = container;
     this.#logger = this.#buildLogger(httpAdapterOptions);
     this.httpAdapterOptions = this.#parseHttpAdapterOptions(httpAdapterOptions);
+    this.#globalPipeList = [];
```

Comment:
Just to be clear: this pipes are applied to every parameter of every handler, right?

Reply:
Yeah, that's the idea